ArcGISRuntimeSDKQt_CppSamples/Geometry/ClipGeometry/ClipGeometry.cpp

```diff
@@ -51,9 +51,7 @@ void ClipGeometry::componentComplete()
 
   // Create a graphics overlay for the Colorado graphic
```

Comment:
Please update comment to match grouping changes, see previous comment for suggestion.